components/sendloop/sources/new-subscriber/new-subscriber.mjs

```diff
@@ -0,0 +1,43 @@
+import common from "../common/base.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  ...common,
+  key: "sendloop-new-subscriber",
+  name: "New Subscriber",
```

Comment:
```suggestion
  name: "New Subscriber Added",
```
I think this would be more in accordance with the source naming guidelines as well, to avoid mistaking this for an action which creates a new subscriber